app/controllers/lawyers_controller.rb

```diff
@@ -0,0 +1,36 @@
+class LawyersController < ApplicationController
+  def index
+    @lawyer = Lawyer.all
+  end
+
+  def create
+    @current_user = current_user
+    @lawyer = current_user.Lawyer.new(lawyer_params)
+    @lawyer.save
+    redirect_to lawyers_path
+  end
+
+  def update
+    @current_user = current_user
+    @lawyer = current_user.Lawyer.new.find(params[:id])
+    @lawyer.update(lawyer_params)
+    redirect_to lawyers_path
+  end
+
+  def show
+    @lawyer = Lawyer.find(params[:id])
+  end
```

Comment:
Action should render JSON:
```suggestion
  def show
    @lawyer = Lawyer.find(params[:id])
    render json: { lawyer: @lawyer }
  end
```